db/versions/0010.yml

```diff
@@ -0,0 +1,379 @@
+version: 10
+migrationScript: 0010-migration.sql
+downgradeScript: 0010-downgrade.sql
+methods:
+  wmworker_pools_entities_load:
+    description: See taskcluster-lib-entities
+    mode: read
+    serviceName: worker_manager
+    args: partition_key text, row_key text
+    returns: table (partition_key_out text, row_key_out text, value jsonb, version integer, etag uuid)
+    body: |-
+      begin
+        return query
+        select
+          wmworker_pools_entities_load.partition_key,
+          'workerPool' as row_key,
+          entity_buf_encode(
+            entity_buf_encode(
+              entity_buf_encode(
+                jsonb_build_object(
+                  'PartitionKey', encode_string_key(worker_pool_id),
```

Comment:
Should this be using `wmworker_pools_entities_load.partition_key,` instead? I guess they both refer to the same thing.

Reply:
Good point -- that would be more efficient.

Reply:
(note that this was the only change I made to the DB migration step; the remainder were to the service migration step)